### ISSUE TITLE
Function name standard

### ISSUE THREAD

**Issue by yrrah2** [ctrlcctrlv/infinity#429]
Many functions use the function_name() format and many others use the functionName() format.
It would be nice if we had an official standard stated about which type we should use as well as many other inconsistencies there is in the scripts of this repo.

**Comment by thebillywayne** [comment]
This seems like a fair place for me to start. I'm starting with ```inc/functions.php```.

**Comment by yrrah2** [comment]
Which format though?

**Comment by thebillywayne** [comment]
`function_name`

Pull request. 
https://github.com/ctrlcctrlv/infinity/pull/433

**Comment by ctrlcctrlv** [comment]
if we pull #433 we lose event_handler and theme compatibility with vichan/tinyboard

let's wait for @czaks to comment

**Comment by czaks** [comment]
i sort of oppose this change. of course, this is for a good reason, but unfortunately, vichan has so many users, that keeping track of this can be hard (i treat infinity as one of vichan users).

what i mostly care about is patch applicability. if we change too much just for the sake of readibility, we happen to have compatibility problems.

in my opinion, this thing should be included incrementally. it's not a secret, that STI's code is shit with regards to standarization. nevertheless, many users depend on it.

take my advice as an upstream advice :^). i also contribute much to 8chan, as you can see, but I believe, this change should happen in upstream first. and for this to happen, a loud and clear message should be sent to our users. also, vichan's upstream is Tinyboard, and many users still use Tinyboard, despite that it's actually dead.

tl;dr: i'm afraid that this patch is likely to cause more problems than benefit.

**Comment by thebillywayne** [comment]
Closed the pull request.

**Comment by czaks** [comment]
btw. i have in plans having a massive vichan rewrite, in ruby, instead of php, which would make a ground for infinity. i have a project in my mind, but it will surely take much time to stabilize and get on par in terms of features, so we are currently locked on this codebase for indefinite time.

**Comment by ctrlcctrlv** [comment]
Closing issue due to @czaks answer.
